server/recceiver/recast.py

```diff
@@ -133,25 +137,29 @@ def recvClientGreeting(self, body):
 
     # 0x0002
     def recvPong(self, body):
-        nonce, = _ping.unpack(body[:_ping.size])
```

Comment:
This looks odd to me, why is it not just
```py
nonce = _ping.unpack(...)
```
Or is the point that it is a list with a single element? I guess then
```py
nonce, *_ = _ping.unpack(...)
```
but maybe this doesn't matter...

Reply:
I think unpack returns a tuple (can actually see in the docs)

Reply:
I though the convention was to prefer `foo,` over `(foo,)`